libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/TriggerConditions/OnAmbigiousIntent.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed under the MIT License.
+// Copyright (c) Microsoft Corporation. All rights reserved.
+
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using Microsoft.Bot.Builder.Dialogs.Adaptive.Recognizers;
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Builder.Dialogs.Adaptive.Conditions
+{
+    /// <summary>
+    /// Actions triggered when an Intent of "AmbigiousIntent" has been emitted by the recognizer.
+    /// </summary>
+    /// <remarks>
+    /// This trigger is run when the utterance has triggered ambiguity between multiple recognizers in a RecognizerSet.
+    /// </remarks>
+    public class OnAmbigiousIntent : OnIntent
+    {
+        [JsonProperty("$kind")]
+        public new const string DeclarativeType = "Microsoft.OnAmbigiousIntent";
```

Comment:
>  public new const string DeclarativeType = "Microsoft.OnAmbigiousIntent"; [](start = 6, length = 74)

You might want to consider adding a property that lets you filter on the intent names involved.